a/Ansible.yml

```diff
@@ -0,0 +1,10 @@
+---
+# This playbook prints a simple debug message
+- name: Echo 
```

Comment:
Inconsistently named with line 8, should it be called `Echo` or `Print`?

Reply:
changed it to print and removed comments too